class RobotoException(Exception):
    pass


class ValidationError(RobotoException):
    pass
